airflow-core/src/airflow/ui/public/i18n/locales/de/dags.json

```
@@ -20,7 +20,8 @@
       "all": "Alle",
       "paused": "Pausiert"
     },
-    "runIdPatternFilter": "Dag Läufe suchen"
+    "runIdPatternFilter": "Dag Läufe suchen",
+    "triggeringUserNameFilter": "Angestoßen von Benutzer suchen"
```

```suggestion
    "triggeringUserNameFilter": "Suche Läufe ausgelöst von..."
```

Ist das zu weit weg vom original? :) nur ein Vorschlag

Nebenfrage: Hast Du auf "angestossen" oder "ausgelöst" standardisiert, ich weiss es grade nicht mehr sorry! 
Frage weil `"triggeringUser": "Benutzer der ausgelöst hat"`. Falls es "angestossen" ist dann "angestossen von" :) 


https://github.com/apache/airflow/blob/938fa15f378abab48d67872def0dab47d46d1850/airflow-core/src/airflow/ui/public/i18n/locales/de/common.json#L59<

---

Ups, ja total korrekt! Ich hatte die Wahl ja extra im README dokumentiert... und war mir nciht mehr sicher und hatte in anderen Übersetzungen geschaut ... jetzt wo Du mich noch mal drauf stößt sehe ich dass ich das bisher inkonsistent hatte. Habe es noch mal überall nachgezuogen "Angestoßen" --> "Ausgelöst". DANKE!